.devops/code-review-pipelines.yml

```
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-2598-redirect-endpoints
```

```suggestion
      ref: main
```
To be replaced after dependent pr merge on eCommerce local